### ISSUE TITLE
All: Breadcrumb item - focus is not visible enough

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#10]
**URL:** https://hcdev.openplus.ca/en
**User Agent**: Chrome/Firefox
**Language**: both

Issue Description: The focus is not visible enough for the breadcrumb items. The focus seems to be on the sides only and top/bottom are missing. 

Please see screenshot of what the focus looks like in Chrome and Firefox.
![image](https://user-images.githubusercontent.com/65728700/126992061-9b598cd5-5cae-4833-b17d-4d2f068e62a8.png)

Steps to reproduce: Tabbing with keyboard
Expected results: The focus should be highly visible.

**Comment by ndolter** [comment]
Validated in both languages, in Chrome and Firefox and the focus has now been corrected.
